components/brave_vpn/browser/connection/win/brave_vpn_wireguard_service/service_utils.cc

```diff
@@ -0,0 +1,99 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_vpn/browser/connection/win/brave_vpn_wireguard_service/service_utils.h"
+
+#include "base/base_paths.h"
+#include "base/command_line.h"
+#include "base/files/file_path.h"
+#include "base/logging.h"
+#include "base/path_service.h"
+#include "brave/components/brave_vpn/browser/connection/win/brave_vpn_wireguard_service/scoped_sc_handle.h"
+#include "brave/components/brave_vpn/browser/connection/win/brave_vpn_wireguard_service/service_constants.h"
+#include "chrome/installer/util/install_service_work_item.h"
+
+namespace brave_vpn {
+
+namespace {
+
+HRESULT HRESULTFromLastError() {
```

Comment:
This function is defined already in several files. Can we extract it to remove the duplicate code?

Reply:
done